assets/js/spotify-api.js

```diff
@@ -0,0 +1,21 @@
+function redirect(url, queryParams) {
+  if (queryParams) {
+    let queryString = "?";
+    for (const key in queryParams) {
+      queryString += `${key}=${queryParams[key]}&`
+    }
+    url += queryString;
+  }
+  location.href = url;
+}
+
+function generatePlaylistCover(e) {
+  const CLIENT_ID = "326af36f3fbd4a0dbcac034ee7216a2d";
```

Comment:
nit: usually single quotes in javascript

Reply:
also should we be putting our `CLIENT_ID` here? this is our spotify developer client id right? should we have this in an `.env` file?

Reply:
not familiar with a `.env` file, can you explain how it would be used in this case? From a quick Google it looks like it's a node module?

Reply:
not a node module necessarily, but I guess if we need a server to source it and this is a free and open id then I buy that it's not worth adding a server just to try and hide it